### PR TITLE
docs: Added operational info

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Added StarkNet operational info.

### Preview URLs

https://starknet-community-libs.github.io/starknet-docs/pr-91/documentation/Ecosystem/ref_operational_info/

### Check List

- [ ] Changes have been done against dev branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91)

### CODE REVIEW

**Review by LucasLvy — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r2, 2 of 2 files at r3, 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91)* status: all files reviewed, 1 unresolved discussion (waiting on @ArielElp, @bbrandtom, and @stoobie)

___
*[`components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc` line 20 at r4](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91#-NFFcwkY4OwHd5ugbzh0:-NFFcwkZ0UnID_ZAHqPr:bmqandr) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/a66d660d88fa43fbe35d07cd066543d6bd653a8d/components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc#L20)):*
> ```AsciiDoc
> == Bridged Tokens
> 
> The `starknet-addresses` repository contains the following `.json` files:
> ```

A table would be much better to display this information imo

**Comment by dan-ziv** [comment]
Your preview build is ready! ✨ Check the following link in 1-2 minutes: https://starknet-community-libs.github.io/starknet-docs/pr-91/documentation/ .

**Review by stoobie — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91)* status: 1 of 6 files reviewed, 1 unresolved discussion (waiting on @ArielElp, @bbrandtom, and @LucasLvy)

___
*[`components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc` line 20 at r4](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91#-NFFcwkY4OwHd5ugbzh0:-NFIIYks1PpntBOk0pIs:b-fzikkd) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/a66d660d88fa43fbe35d07cd066543d6bd653a8d/components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc#L20)):*
<details><summary><i>Previously, LucasLvy (Lucas) wrote…</i></summary><blockquote>

A table would be much better to display this information imo
</blockquote></details>

@lucaslvy Which info, the list of parameters?

Take a look at the preview to see how it appears: https://starknet-community-libs.github.io/starknet-docs/pr-91/documentation/Ecosystem/ref_operational_info/

Did you see my inline questions?

**Comment by dan-ziv** [comment]
Your preview build is ready! ✨ Check the following link in 1-2 minutes: https://starknet-community-libs.github.io/starknet-docs/pr-91/documentation/ .

**Review by LucasLvy — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91)* status: 1 of 6 files reviewed, 1 unresolved discussion (waiting on @ArielElp, @bbrandtom, and @stoobie)

___
*[`components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc` line 20 at r4](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91#-NFFcwkY4OwHd5ugbzh0:-NFIhvIX76eJDS5pClWE:b-g1wpog) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/a66d660d88fa43fbe35d07cd066543d6bd653a8d/components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc#L20)):*
<details><summary><i>Previously, stoobie (Steve Goodman) wrote…</i></summary><blockquote>

@lucaslvy Which info, the list of parameters?

Take a look at the preview to see how it appears: https://starknet-community-libs.github.io/starknet-docs/pr-91/documentation/Ecosystem/ref_operational_info/

Did you see my inline questions?
</blockquote></details>

A table to display everything on this page and not link to the json file

File: components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[id="operational_info"]
+= Operational info
+
+==  StarkNet Alpha on Mainnet
+
+The StarkNet core contract:: link:https://etherscan.io/address/0xc662c410C0ECf747543f5bA90660f6ABeBD9C8c4[`0xc662c410C0ECf747543f5bA90660f6ABeBD9C8c4`^]
+Verifier address::  link:https://etherscan.io/address/0x47312450B3Ac8b5b8e247a6bB6d523e7605bDb60[`0x47312450B3Ac8b5b8e247a6bB6d523e7605bDb60`^]
+// ??? This URL does not work. What is the correct URL???
```
Comment (LucasLvy) [inline components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc:8]: Yes it's a base url for the api routes

File: components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Sequencer URL:: https://alpha-mainnet.starknet.io[https://alpha-mainnet.starknet.io^]
+
+== StarkNet Alpha version 4 on Goerli
+
+The StarkNet core contract:: link:https://goerli.etherscan.io/address/0xde29d060D45901Fb19ED6C6e959EB22d8626708e[`0xde29d060D45901Fb19ED6C6e959EB22d8626708e`^]
+Verifier address::  link:https://goerli.etherscan.io/address/0xAB43bA48c9edF4C2C4bB01237348D1D7B28ef168[`0xAB43bA48c9edF4C2C4bB01237348D1D7B28ef168`^]
+// ??? This URL does not work. What is the correct URL???
```
Comment (LucasLvy) [inline components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc:15]: Yes it's a base url for the api routes

Comment (stoobie) [reply components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc:15]: PTAL

**Review by bbrandtom — APPROVED** [review]
:lgtm:

> Reviewed 2 of 8 files at r9, 2 of 6 files at r10, 3 of 3 files at r11, 1 of 1 files at r12, 1 of 2 files at r13, 1 of 1 files at r14, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91#-:-NFbgeD12PuHRRaaXgvC:bnfp4nl)* status: all files reviewed, 3 unresolved discussions (waiting on @ArielElp, @LucasLvy, and @stoobie)

**Review by LucasLvy — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91#-:-NFdDsHiDIE27DbyrP6V:bnfp4nl)* status: all files reviewed, 3 unresolved discussions (waiting on @ArielElp and @stoobie)

**Comment by dan-ziv** [comment]
Your preview build is ready! ✨ Check the following link in 1-2 minutes: https://starknet-community-libs.github.io/starknet-docs/pr-91/documentation/ .

**Review by stoobie — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91)* status: 2 of 10 files reviewed, 3 unresolved discussions (waiting on @ArielElp, @bbrandtom, and @LucasLvy)

___
*[`components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc` line 20 at r4](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91#-NFFcwkY4OwHd5ugbzh0:-NFhHRE67tX6UaSc0TYl:b-e6rw5z) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/a66d660d88fa43fbe35d07cd066543d6bd653a8d/components/StarkNet/modules/ROOT/pages/Ecosystem/ref_operational_info.adoc#L20)):*
<details><summary><i>Previously, LucasLvy (Lucas) wrote…</i></summary><blockquote>

A table to display everything on this page and not link to the json file
</blockquote></details>

Doing so creates a maintenance problem. I don't own the json file. If it's updated without me knowing, then this page that we're working on will become obsolete.

**Review by glrn — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91#-:-NFhKBQS0M2anYmKicGY:bnfp4nl)* status: 2 of 10 files reviewed, 3 unresolved discussions (waiting on @ArielElp, @bbrandtom, and @LucasLvy)

**Review by stoobie — COMMENTED** [review]
> Reviewed 1 of 2 files at r13, 8 of 8 files at r15, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91)* status: all files reviewed, 3 unresolved discussions (waiting on @ArielElp, @bbrandtom, and @LucasLvy)

**Review by glrn — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/91#-:-NFiIuII2MTbILHGhPSZ:bnfp4nl)* status: all files reviewed, 3 unresolved discussions

**Pull request merged**